docs/source/howtos/get_resolution_function.rst

```diff
@@ -0,0 +1,72 @@
+How To Get the Resolution Function
+==================================
+
+The short answer for getting the :term:`resolution function` of a particular
+:term:`version` of a given :term:`instrument` is to use the
+:py:meth:`resolution_functions.instrument.Instrument.get_resolution_function`
+method, e.g.:
+
+>>> from resolution_functions import Instrument
+>>> tosca = Instrument('TOSCA')
+>>> book = tosca.get_resolution_function('book', detector_bank='Forward')
+>>> print(book)
+ToscaBookModel(citation="")
```

Comment:
Are we doing any doctest magic to keep this up-to-date? We need to remember to update this citation when it is in the main code.

Reply:
Sure. Feel free to tag it "nice to have" and beyond scope of this rotation project!